app/config/services/platform.xml

```diff
@@ -371,12 +371,20 @@
         </service>
 
         <!-- CitizenActions -->
-        <service id="AppBundle\CitizenAction\CitizenActionManager">
-            <argument type="service" id="app.repository.citizen_action"/>
-        </service>
+        <service autowire="true" id="AppBundle\CitizenAction\CitizenActionManager"/>
 
         <service autowire="true" id="AppBundle\CitizenAction\CitizenActionCommandHandler" />
 
+        <service autoconfigure="true" id="AppBundle\CitizenAction\Voter\OrganizerCitizenActionVoter" />
```

Comment:
This service is already declared with :
 `<prototype namespace="AppBundle\CitizenAction\Voter\" resource="../../../src/CitizenAction/Voter/" />` in `services.xml`